packages/main/src/RPA/JSON.py

```diff
@@ -24,21 +109,43 @@ def __init__(self):
 
     @keyword("Load JSON from file")
     def load_json_from_file(self, filename: str) -> Any:
-        """Load JSON data from a file.
+        """Load JSON data from a file, and return it as JSON object.
+        Depending on the input file the object can be either a dictionary,
+        a list, or a scalar value.
+
+        :param filename: path to input file
+
+        Example:
+
+        .. code:: robotframework
+
+           &{auth}=    Load JSON from file    auth.json
+           Log   Current auth token: ${auth.token}
 
-        :param filename: path to file
-        :return: json as a dictionary object
         """
         self.logger.info("Loading JSON from file: %s", filename)
         with open(filename) as json_file:
             return json.load(json_file)
 
     @keyword("Save JSON to file")
     def save_json_to_file(self, doc: Any, filename: str) -> None:
-        """Save JSON object into a file.
+        """Save a JSON object, or a string containg a JSON value, into a file.
```

Comment:
Is it a "JSON object" or an ordinary Python dictionary...?

Reply:
See above about JSONs. Technically "JSON serializable object" I guess?